app/code/Magento/CatalogUrlRewrite/Test/Mftf/ActionGroup/StorefrontAssertProductRewriteUrlSubCategoryActionGroup.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontAssertProductRewriteUrlSubCategoryActionGroup">
```

Comment:
Hi @engcom-Echo. Could I ask you to rename the action group in order to follow the general naming rules, please? So we have

`AssertStorefrontProductRewriteUrlSubCategoryActionGroup`

in the result.

Please, check the [technical guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html) (part 11.3.5).

Thank you.

Reply:
Thank you